com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/delegates/dependency/CopybookDependencyService.java

```diff
@@ -24,11 +26,14 @@
  * management
  */
 public interface CopybookDependencyService {
+
+  void addCopybookInDepFile(RequiredCopybookEvent event, String requiredCopybookName);
+
+  boolean isFileInDidOpen(RequiredCopybookEvent event);
```

Comment:
This is an exposure of implementation details to the public interface. Not critical, just bad practice.

Reply:
It was a temporary solution waiting for Powermock since I need to use this method in unit test.